ush/load_ufsda_modules.sh

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env bash
 
 ###############################################################
-if [[ "${DEBUG_WORKFLOW:-NO}" == "NO" ]]; then
+if [[ "${DEBUG_WORKFLOW}" == "NO" ]]; then
```

Comment:
I think this was a mistake to merge this change in...  did anyone test? @jiaruidong2017 seems to be having an issue here